core/src/main/scala/kanaloa/reactive/dispatcher/Dispatcher.scala

```diff
@@ -55,12 +55,15 @@ object Dispatcher {
   case class Settings(
     workTimeout:     FiniteDuration                 = 1.minute,
     workRetry:       Int                            = 0,
+    updateInterval:  FiniteDuration                 = 1.second,
     dispatchHistory: DispatchHistorySettings,
     workerPool:      ProcessingWorkerPoolSettings,
     circuitBreaker:  Option[CircuitBreakerSettings],
     backPressure:    Option[BackPressureSettings],
     autoScaling:     Option[AutoScalingSettings]
-  )
+  ) {
+    val metricsCollectorSettings = MetricsCollector.MetricsCollectorSettings(updateInterval)
```

Comment:
Why is this not configurable like all of the other Settings?  Shouldn't this be sourceable from the Configuration?